src/Atoms/Card/Card.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+import BaseCard from '@material-ui/core/Card'
+
+import { FlexBox } from '../../Templates'
+
+const Card = ({
+  children,
+  ...props
+}) => (
+  <FlexBox gap={2} component={BaseCard} {...props} bgcolor="grey.100" p={2} borderRadius="borderRadius" flexDirection="column">
+    {children}
+  </FlexBox>
+)
```

Comment:
Props should be passed last to make defaults overridable.
Then, no need to extract children as you pass down props anyway.
So you can simplify this :
```suggestion
const Card = props => <FlexBox
  component={BaseCard}
  gap={2}
  p={2}
  bgcolor="grey.100"
  borderRadius="borderRadius"
  flexDirection="column"
  {...props}
/>
```